app/controllers/france_connect/particulier_controller.rb

```diff
@@ -11,11 +11,8 @@ def callback
     fetched_fc_information = FranceConnectService.retrieve_user_informations_particulier(params[:code])
 
     france_connect_information = FranceConnectInformation
-      .find_by(france_connect_particulier_id: fetched_fc_information[:france_connect_particulier_id])
-
-    if france_connect_information.nil?
-      fetched_fc_information.save
-      france_connect_information = fetched_fc_information
+      .find_or_create_by(france_connect_particulier_id: fetched_fc_information[:france_connect_particulier_id]) do |fci|
+      fci.assign_attributes(fetched_fc_information.serializable_hash)
```

Comment:
Sauter une ligne avant celle-ci pour améliorer la lisibilité

Reply:
On perd le `save` dans l'affaire, non ?

Reply:
le save est build in dans le find_or_create_by.
sauter une ligne sous le do ?

Reply:
yes, sous le do